domains/moku.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Portfolio Website",
+  "repo": "https://github.com/Moku922/CSS_HTML-CV_Website",
+  "owner": {
+    "username": "Moku922",
+    "email": "abdulmoqtadir1@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://moku922.github.io/CSS_HTML-CV_Website/"
```

CNAME records can't contain protocol or `/`
```suggestion
    "CNAME": "moku922.github.io"
```